debian/rules

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE=1
+
+export DH_OPTIONS
+
+export DH_GOPKG := github.com/awslabs/amazon-ecr-credential-helper
+
+VERSION := $(shell cat VERSION)
+GITCOMMIT_SHA := $(shell git rev-parse --short HEAD)
+package_root := "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
+
+%:
+	dh $@ --buildsystem=golang --with=golang
+
+override_dh_auto_build:
+	dh_auto_build -- -ldflags "-X $(package_root)/version.Version=$(VERSION) -X $(package_root)/version.GitCommitSHA=$(GITCOMMIT_SHA)"
+	mkdir -p bin/local
+	cp obj-$(DEB_HOST_GNU_TYPE)/bin/docker-credential-ecr-login bin/local
```

Comment:
Do you actually need to be doing any of the stuff on lines 18-19?

Reply:
Can I use `obj-$(DEB_HOST_GNU_TYPE)` in `amazon-ecr-credential-login.install`?  If so, I don't need these lines.

Reply:
You can't. My (incomplete) suggestion was more to override the .dh_auto_install` rule and install the build artifacts into the package in a custom rule. It seems somewhat strange to copy the files to a new location just so you can move them out of it.

Reply:
Should I do that instead of using `amazon-ecr-credential-login.install`?